src/main/java/net/dv8tion/jda/api/JDABuilder.java

```diff
@@ -216,25 +217,26 @@ public JDABuilder setContextEnabled(boolean enable)
     }
 
     /**
-     * Enable stream-compression on the gateway connection,
+     * Enable compression on the gateway connection,
      * this will decrease the amount of used bandwidth for the running bot instance
      * for the cost of a few extra cycles for decompression.
-     * <br><b>Default: true</b>
+     * <br><b>Default: {@link net.dv8tion.jda.api.utils.Compression#ZLIB}</b>
      *
      * <p><b>We recommend to keep this enabled unless you have issues with the decompression</b>
      * <br>This mode might become obligatory in a future version, do not rely on this switch to stay.
```

Comment:
This comment is effectively useless. Lets nix it.

Reply:
I don't see how it is useless, they might still require a specific compression in the future.